<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImsClass</class>
 <widget class="QMainWindow" name="ImsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ims</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Binarization Threshold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="binThreshEdit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>285</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionLoadBackground"/>
    <addaction name="actionLoadImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string extracomment="Load a black and white calibration background image"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadBackground"/>
   <addaction name="actionLoadImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadBackground">
   <property name="text">
    <string>Background</string>
   </property>
   <property name="toolTip">
    <string>Load a B/W background (calibration) image</string>
   </property>
  </action>
  <action name="actionLoadImage">
   <property name="text">
    <string>Image</string>
   </property>
   <property name="toolTip">
    <string>Load a raw image, color or B/W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ims.qrc"/>
 </resources>
 <connections/>
</ui>
